webbpsf/trending.py

```diff
@@ -56,12 +58,19 @@ def wavefront_time_series_plot(opdtable, start_date=None, end_date=None, label_v
 
     rmses = []
 
+    if 'rms_wfe' in opdtable.colnames:
+        rmses = opdtable['rms_wfe']
+
     dates = astropy.time.Time(opdtable['date'], format='isot')
     pre_or_post = []
 
     for row in opdtable:
-        full_file_path = os.path.join(webbpsf.utils.get_webbpsf_data_path(), 'MAST_JWST_WSS_OPDs', row['fileName'])
-        rmses.append(fits.getheader(full_file_path, ext=1)['RMS_WFE'])
+        if os.path.isfile(row['fileName']) is False:
+            full_file_path = os.path.join(webbpsf.utils.get_webbpsf_data_path(), 'MAST_JWST_WSS_OPDs', row['fileName'])
+        else:
+            full_file_path = row['fileName']
+        if len(rmses) == 0:
+            rmses.append(fits.getheader(full_file_path, ext=1)['RMS_WFE'])
```

Comment:
I think this part may be problematic in the `len(rmses) == 0` check. Won't that make it just append the calculated RMS for the first row only, and then after that the length isn't 0, so it won't append for any of the rest of the lines in the table. 

I suggest instead `if 'rms_wfe' not in opdtable.colnames` so it's consistent with  but opposite from the check above, and like the other version of this check in line 219.

Reply:
Agreed I will make this change.  Thanks for the suggestion.